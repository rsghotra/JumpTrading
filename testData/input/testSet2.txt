0, 100000, 0, 2, 1075
0, 100001, 0, 3, 1080
0, 100002, 0, 100, 975
0, 100003, 0, 5, 1080
0, 100004, 1, 10, 1078
1, 100002
0, 100005, 0, 10, 1078
0, 100006, 1, 100, 1072
