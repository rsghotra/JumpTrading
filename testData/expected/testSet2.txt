2,3,1080
4,100004,7
3,100001
2,5,1080
4,100004,2
3,100003
2,2,1078
4,100005,8
3,100004
2,8,1078
4,100006,92
3,100005
2,2,1075
4,100006,90
3,100000